individual/tests/graphql_mutation_group_test.py

```diff
@@ -474,7 +474,7 @@ def test_add_individual_to_group_row_security(self):
         )
         content = json.loads(response.content)
         internal_id = content['data']['addIndividualToGroup']['internalId']
-        self.assert_mutation_error(internal_id, _('unauthorized.location'))
+        self.assert_mutation_success(internal_id)
```

Comment:
@delcroip why has this been changed to success? Shouldn't we prevent such a scenario to keep location data consistent between individual and their group?